### DISCUSSION TITLE
cefsharp not show network shared folder

### DISCUSSION THREAD

**Discussion by wnunezc** [cefsharp/CefSharp#3501] [Q&A]
cefsharp not show network shared folder.

When opening the files dialog, to upload a file it does not show the folders, files or Windows Shared Disks, how can I enable it?

**Current dialog:**
![image](https://user-images.githubusercontent.com/1207306/114590228-1ba34880-9c4e-11eb-9aec-5560b0a6ee6f.png)

**Windows Explorer**
![image](https://user-images.githubusercontent.com/1207306/114590261-2362ed00-9c4e-11eb-97c7-f945fcc111f6.png)

**Comment by amaitland** [comment]
Test with the CEF Sample application see https://github.com/cefsharp/CefSharp/blob/master/CONTRIBUTING.md#cefsharp-vs-chromium-embedded-frameworkcef

If the problem reproduces with cefclient then you'll have to ask on https://magpcss.org/ceforum/index.php

**Reply by wnunezc** [reply]
the problem is related to elevated privileges ... with a normal work fine but I have another problem, if I create an update system I cannot update the application folder in C:\Program Files (x86)\Application test... fix something and the other breaks XD

**Reply by amaitland** [reply]
Use a separate process to perform the update. This is a pretty generic problem, not CefSharp specific,  try asking on stackoverflow if you need help.

**Reply by wnunezc** [reply]
i am working on it: https://stackoverflow.com/questions/67081967/access-denied-to-appdomain-currentdomain-basedirectory
